frontend/packages/patternfly/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export * from './notification-drawer';
```

Comment:
We should check this re the issue with circular dependencies in slack yesterday.  One of our tools struggled.

Reply:
(But I see you removed other re-exports)